packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -130,13 +130,13 @@ class RenderListWheelViewport
   /// An arbitrary but aesthetically reasonable default value for [diameterRatio].
```

Comment:
while you're here, change the `1-`, `2-`, etc, to `1.`, `2.`, to make markdown work better.

Reply:
Done